2014-05-26 15:41:31.856 GMT Thread[main,5,main] Cleanup action starting
java.sql.SQLException: Failed to start database 'jar:(thermolab.jar)thermolabdb', see the next exception for details.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at br.com.vrtech.thermolab.DataBase.init(DataBase.java:129)
	at br.com.vrtech.thermolab.DataBase.getInstance(DataBase.java:73)
	at br.com.vrtech.thermolab.DataBase.getInstance(DataBase.java:62)
	at br.ufrgs.enq.gbflores.rcm.ResidueCurveSquare.main(ResidueCurveSquare.java:54)
Caused by: java.sql.SQLException: Failed to start database 'jar:(thermolab.jar)thermolabdb', see the next exception for details.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 16 more
Caused by: java.sql.SQLException: Java exception: 'thermolab.jar (No such file or directory): java.io.FileNotFoundException'.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.javaException(Unknown Source)
	... 13 more
Caused by: java.io.FileNotFoundException: thermolab.jar (No such file or directory)
	at java.util.zip.ZipFile.open(Native Method)
	at java.util.zip.ZipFile.<init>(ZipFile.java:215)
	at java.util.zip.ZipFile.<init>(ZipFile.java:145)
	at java.util.zip.ZipFile.<init>(ZipFile.java:159)
	at org.apache.derby.impl.io.JarStorageFactory.doInit(Unknown Source)
	at org.apache.derby.impl.io.BaseStorageFactory.init(Unknown Source)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.privGetStorageFactoryInstance(Unknown Source)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.access$400(Unknown Source)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$11.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.getCanonicalServiceName(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.findProviderAndStartService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startPersistentService(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.startPersistentService(Unknown Source)
	... 13 more
============= begin nested exception, level (1) ===========
java.sql.SQLException: Java exception: 'thermolab.jar (No such file or directory): java.io.FileNotFoundException'.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.javaException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at br.com.vrtech.thermolab.DataBase.init(DataBase.java:129)
	at br.com.vrtech.thermolab.DataBase.getInstance(DataBase.java:73)
	at br.com.vrtech.thermolab.DataBase.getInstance(DataBase.java:62)
	at br.ufrgs.enq.gbflores.rcm.ResidueCurveSquare.main(ResidueCurveSquare.java:54)
Caused by: java.sql.SQLException: Java exception: 'thermolab.jar (No such file or directory): java.io.FileNotFoundException'.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 16 more
Caused by: java.io.FileNotFoundException: thermolab.jar (No such file or directory)
	at java.util.zip.ZipFile.open(Native Method)
	at java.util.zip.ZipFile.<init>(ZipFile.java:215)
	at java.util.zip.ZipFile.<init>(ZipFile.java:145)
	at java.util.zip.ZipFile.<init>(ZipFile.java:159)
	at org.apache.derby.impl.io.JarStorageFactory.doInit(Unknown Source)
	at org.apache.derby.impl.io.BaseStorageFactory.init(Unknown Source)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.privGetStorageFactoryInstance(Unknown Source)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.access$400(Unknown Source)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$11.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.getCanonicalServiceName(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.findProviderAndStartService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startPersistentService(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.startPersistentService(Unknown Source)
	... 13 more
============= end nested exception, level (1) ===========
============= begin nested exception, level (2) ===========
java.sql.SQLException: Java exception: 'thermolab.jar (No such file or directory): java.io.FileNotFoundException'.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.javaException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at br.com.vrtech.thermolab.DataBase.init(DataBase.java:129)
	at br.com.vrtech.thermolab.DataBase.getInstance(DataBase.java:73)
	at br.com.vrtech.thermolab.DataBase.getInstance(DataBase.java:62)
	at br.ufrgs.enq.gbflores.rcm.ResidueCurveSquare.main(ResidueCurveSquare.java:54)
Caused by: java.io.FileNotFoundException: thermolab.jar (No such file or directory)
	at java.util.zip.ZipFile.open(Native Method)
	at java.util.zip.ZipFile.<init>(ZipFile.java:215)
	at java.util.zip.ZipFile.<init>(ZipFile.java:145)
	at java.util.zip.ZipFile.<init>(ZipFile.java:159)
	at org.apache.derby.impl.io.JarStorageFactory.doInit(Unknown Source)
	at org.apache.derby.impl.io.BaseStorageFactory.init(Unknown Source)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.privGetStorageFactoryInstance(Unknown Source)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.access$400(Unknown Source)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$11.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.getCanonicalServiceName(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.findProviderAndStartService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startPersistentService(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.startPersistentService(Unknown Source)
	... 13 more
============= end nested exception, level (2) ===========
============= begin nested exception, level (3) ===========
java.io.FileNotFoundException: thermolab.jar (No such file or directory)
	at java.util.zip.ZipFile.open(Native Method)
	at java.util.zip.ZipFile.<init>(ZipFile.java:215)
	at java.util.zip.ZipFile.<init>(ZipFile.java:145)
	at java.util.zip.ZipFile.<init>(ZipFile.java:159)
	at org.apache.derby.impl.io.JarStorageFactory.doInit(Unknown Source)
	at org.apache.derby.impl.io.BaseStorageFactory.init(Unknown Source)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.privGetStorageFactoryInstance(Unknown Source)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.access$400(Unknown Source)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$11.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.getCanonicalServiceName(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.findProviderAndStartService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startPersistentService(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.startPersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at br.com.vrtech.thermolab.DataBase.init(DataBase.java:129)
	at br.com.vrtech.thermolab.DataBase.getInstance(DataBase.java:73)
	at br.com.vrtech.thermolab.DataBase.getInstance(DataBase.java:62)
	at br.ufrgs.enq.gbflores.rcm.ResidueCurveSquare.main(ResidueCurveSquare.java:54)
============= end nested exception, level (3) ===========
Cleanup action completed
